### ISSUE TITLE
More EUR support

### ISSUE THREAD

**Issue by igorvanderlinden** [tubededentifrice/streamdeck-crypto#4]
Hey Tubededentifrice!

First of all, I absolutely love your plugin for the crypto prices :)
However, BitFinex does not support many EUR rates.
I would love to have LINK-EUR, LTC-EUR, XRP-EUR and XLM-EUR.
Currently of my main watched coins I can only use ETH-EUR and BTC-EUR.

As I'm from Europe, this is a bit disappointing, as your plugin is perfect, but it just doesn't support the conversion rates I want to watch. Do you have any suggestions for me to make it work, or is it possible for you to get conversion rates from other exchanges in your plugin? Like Coinbase or Binance?

What I did now to watch the rates is using a multiplier of 0,82 (as 1 USD is about 0,82 EUR), but as you understand this is not ideal, as the EUR-USD rate does change quite a bit every day and this won't be taken into account by the plugin's multiplier. Maybe it's possible to write a code for having a real-time EUR-USD or USD-EUR multiplier?

Also, the background color rule does only support the main rate (without multiplier), so everytime I set background rule values, I should also convert those to USD to be able to make it work for just a little bit.

I love your work and see the potential, also in Europe and when you are trading with EUR, but for me it's not fully functional in an ideal situation yet. It would be awesome if you could try implementing other exchanges (and with that other exchange rates), or a way to have the multiplier change in real time too! :)

Thankyou in advance, and awaiting your answer!
Igor

**Comment by tubededentifrice** [comment]
Hi there
Here we go https://github.com/tubededentifrice/streamdeck-crypto/commit/ed3c73fecb170e1f7da661a3bc85d323928d2d52

I've added a currency converter drop down in the settings that should handle all conversions. The resulting values are also properly used in various functions.

It should be updated in the store soon, or you can download it from https://github.com/tubededentifrice/streamdeck-crypto/tree/master/Release if you can't wait

**Comment by BeyondRico** [comment]
Hi @tubededentifrice
When using the convert  to option it switches back to the selected crypto pair after 2 sec.
Am I missing something or is this a bug?

**Comment by tubededentifrice** [comment]
@BeyondRico Probably a bug as it should be straightforward to use. Could you describe the behavior with more details and/or screenshots if possible, so I can repro it?

Thanks.

**Comment by BeyondRico** [comment]
@tubededentifrice it no longer occurs after I restarted the computer. But when I set it up it changed back to the standard selected crypto pair (usd in my case, when trying to convert to Euro). The strange part was that when I clicked on the button to show the graph and clicked again it showed the converted price(Euro), it then switched back to usd after the price changed. But as mentioned it does not happen anymore, I am not sure what fixed the problem.
